astroquery/esa/xmm_newton/core.py

```diff
@@ -266,5 +267,80 @@ def get_columns(self, table_name, *, only_names=True, verbose=False):
+            The radius to query the target in degrees
+        Returns
+        -------
+        Four astropy.table.table.Table containing the metadata
+        of the given target
+        """
+        if not target_name and not coordinates:
+                raise Exception(("Input parameters needed, "
+                                 "please provide the name "
+                                 "or the coordinates of the target"))
+
+        epic_source = {"table": "xsa.v_epic_source",
+                       "column": "epic_source_equatorial_spoint"}
+        cat_4xmm = {"table": "xsa.v_epic_source_cat",
+                    "column": "epic_source_cat_equatorial_spoint"}
+        stack_4xmm = {"table": "xsa.v_epic_xmm_stack_cat",
+                      "column": "epic_stack_cat_equatorial_spoint"}
+        slew_source = {"table": "xsa.v_slew_source_cat",
+                       "column": "slew_source_cat_equatorial_spoint"}
+
+        cols = "*"
+
+        c = coordinates
+        if not coordinates:
+            c = SkyCoord.from_name(target_name, parse=True)
+
+        if type(c) is not SkyCoord:
+            raise Exception(("The coordinates must be an "
+                             "astroquery.coordinates.SkyCoord object"))
+        if not radius:
+            radius = 0.1
+
+        query_fmt = ("select %s from %s "
+                     "where 1=contains(%s, circle('ICRS', %f, %f, %f));")
+        epic_source_table = self.query_xsa_tap(query_fmt % (cols,
+                                               epic_source["table"],
+                                               epic_source["column"],
+                                               c.ra.degree,
+                                               c.dec.degree,
+                                               radius))
+        cat_4xmm_table = self.query_xsa_tap(query_fmt % (cols,
+                                            cat_4xmm["table"],
+                                            cat_4xmm["column"],
+                                            c.ra.degree,
+                                            c.dec.degree,
+                                            radius))
+        stack_4xmm_table = self.query_xsa_tap(query_fmt % (cols,
+                                              stack_4xmm["table"],
+                                              stack_4xmm["column"],
+                                              c.ra.degree,
+                                              c.dec.degree,
+                                              radius))
+        slew_source_table = self.query_xsa_tap(query_fmt % (cols,
+                                               slew_source["table"],
+                                               slew_source["column"],
+                                               c.ra.degree,
+                                               c.dec.degree,
+                                               radius))
+        return epic_source_table, \
```

Comment:
no need for the linebreaks, lines up to 100 char is perfectly fine, and up-to 120 is OK